app/controllers/application_controller/miq_request_methods.rb

```diff
@@ -4,6 +4,18 @@ module ApplicationController::MiqRequestMethods
     helper_method :dialog_partial_for_workflow
   end
 
+  def get_provisioning_request_source(src_id, src_type_string)
```

Comment:
@syncrou Of course, I'm just being expedient while I stumble through this process.